src/main/java/de/retest/recheck/Properties.java

```diff
@@ -10,8 +15,11 @@
 	public static final String PROPERTY_VALUE_SEPARATOR = ";";
 	public static final String ZIP_FOLDER_SEPARATOR = "/";
 
-	public static final double WINDOW_MATCH_THRESHOLD = 0.8;
-	public static final double WINDOW_CHILDS_MATCH_THRESHOLD = 0.5;
+	public static final double WINDOW_MATCH_THRESHOLD_DEFAULT = 0.8;
+	public static final String WINDOW_MATCH_THRESHOLD_PROPERTY = "de.retest.recheck.windowMatchThreshold";
```

Comment:
We do not have a window per se. Should we rename this to `ROOT_ELEMENT_THRESHOLD_PROPERTY`?

Reply:
Done